container.go

```diff
@@ -834,7 +834,7 @@ func (c *Container) hotplugDrive() error {
 	}).Info("Block device detected")
```

Comment:
Related to the commit message. Typo /idenitfy/identify/

Reply:
thanks - fixed....